src/tables/STAT.js

```diff
@@ -0,0 +1,54 @@
+import r from 'restructure';
+
+
+let AxisRecord = new r.Struct({
+  axisTag:      new r.String(4),    // e.g. 'ital', 'wght'
+  axisNameID:   r.uint16,
+  axisOrdering: r.uint16,
+})
+
+let AxisValue = new r.VersionedStruct( r.uint16, {
+  header: {
+    axisIndex:      r.uint16,
+    flags:          r.uint16,
+    valueNameID:    r.uint16,
+  },
+  1: {
+    value:          r.fixed32be,
```

Comment:
Everything is assumed to be big endian by default. `r.fixed32` should work and is an alias for `r.fixed32be`